bazel/copts.bzl

```diff
@@ -0,0 +1,19 @@
+copts_common = [
+    "-Werror",
+]
+
+copts_ = copts_common + select({
```

Comment:
Thinking through this one final time I have one last request which is that we actually make this a function, i.e., `def copts():`.

Here's the reasoning: I can imagine following this pattern other places too, i.e., when we have shared configuration, however, I can imagine that in some circumstances we'll actually want to "compute" the configuration versus just declare it. While these are almost one in the same with Bazel, the added flexibility of being able to just come and update a function seems worth starting that pattern this way from the start rather than bolting it on later, even if most of our shared configurations will be declarative like this one. Make sense?

Reply:
Yep, great idea, as usual:)